### PR TITLE
Set up sbt-ci-release

### PR DESCRIPTION
**Pull request by eed3si9n**
Here's my attempt at setting up automated release via Travis CI.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,22 +1,37 @@
 language: scala
-matrix:
+
+stages:
+  - name: test
+  - name: release
+    if: (tag IS present) AND NOT fork
```
Comment (nevillelyh) [inline .travis.yml:6]: So `version.sbt` is no more and we only rely on tags? Are we suppose to tag manually for release and push?

Comment (eed3si9n) [reply .travis.yml:6]: That's right. You would create a tag starting with `v` and the plugin is supposed to pick it up and publish to Bintray.

**Comment by eed3si9n** [comment]
I'm currently testing similar set up on sbt-buildinfo. Please hold the merge.

**Comment by eed3si9n** [comment]
ok. I was able to release sbt-buildinfo 0.10.0-M4 - https://travis-ci.org/github/sbt/sbt-buildinfo/jobs/715990933

**Review by RustedBones — APPROVED** [review]

**Pull request merged**